docs/publishing.md

```diff
@@ -1,37 +1,46 @@
 
-## Before you begin
+The release process is mostly automated using two GitHub workflows:
 
-Any project owner (or release designee) can complete the following steps to publish a new version.
-
-The steps in this process assume:
-
-- `NPM_TOKEN` is set in the repository secrets. If the token is invalidated or unset, a member of the `@mdn` organization on npm must [create a new token](https://docs.npmjs.com/creating-and-viewing-authentication-tokens) and [add it to the repository's secrets](https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#creating-encrypted-secrets-for-a-repository).
-- You have cloned the mdn/browser-compat-data repository and you have the latest `main` branch checked out.
-- You have permissions to merge pull requests and publish releases.
-- You have [the `gh` CLI](https://cli.github.com/) installed.
-- You have [`jq`](https://stedolan.github.io/jq/) installed.
+1. The [`release-pr` workflow](https://github.com/mdn/browser-compat-data/blob/main/.github/workflows/release-pr.yml) manages a pull request draft that bumps the `package.json` version and updates the release notes.
+2. The [`release` workflow](https://github.com/mdn/browser-compat-data/blob/main/.github/workflows/release.yml) creates the GitHub release and tag, and publishes the release on NPM.
 
 ## Performing a release
 
-Releasing a new version of BCD is a two-step process: first, the release script is run to bump version number, generate the changelog, and create a pull request; then, once the pull request is merged, a release on GitHub is performed.
+A release is usually performed by merging the pull request managed by the `release-pr` workflow:
 
-First, run `npm run release`. This will automatically perform all of the steps needed to release a new version of BCD. If user input is needed, the script will announce as such. Once the script is complete, a pull request to release a new version of BCD will be created from your GitHub account, which can be reviewed and merged.
+1. The merge commit will trigger the `create-release` job of the `release` workflow.
+2. The `create-release` job publishes a [GitHub release](https://github.com/mdn/browser-compat-data/releases) and [tag](https://github.com/mdn/browser-compat-data/tags) based on the updated `package.json` and release notes.
+3. The published GitHub release will trigger the `publish-release` job of the `release` workflow.
+4. The `publish-release` job performs the following steps:
+   - Builds the release.
+   - Publishes the release to NPM.
+   - Adds the `data.json` as an asset to the GitHub release.
+   - Dispatches a `bcd_release` event on the [mdn/bcd-utils](https://github.com/mdn/bcd-utils) repository, which triggers the deployment of the new BCD release on MDN.
```

Comment:
As far as I can tell, none of this is something the reader _does_ to perform a release. It should all go in the "Behind the scenes" section.

Reply:
Good point, I've moved those details now.